html/changelogs/example.yml

```diff
@@ -21,16 +21,17 @@
 #   tgs (TG-ported fixes?)
 #################################
 
-# Your name.  
+# Your name.
 author: N3X15
 
 # Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
 delete-after: True
 
 # Any changes you've made.  See valid prefix list above.
-# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# USE SPACES.  NOT TABS.  SPACES.
 # SCREW THIS UP AND IT WON'T WORK.
```

Comment:
Make sure people are aware you need - SPACE rscadd etc for it to work  somewhere or another